easybuild/easyconfigs/a/assembly-stats/assembly-stats-1.0.1_fix_str_cast.patch

```diff
@@ -0,0 +1,39 @@
+diff -ruN assembly-stats-original/fasta_unittest.cpp assembly-stats/fasta_unittest.cpp
```

Comment:
We like to ensure that all patches include a tiny description of why this is needed/what it does + the author
(when you update the patch, remember to also update the new checksum)